### PR TITLE
Add s390x to node images

### PR DESCRIPTION
**Pull request by SimenB**
See https://github.com/nodejs/docker-node/pull/490

### CODE REVIEW

File: library/node [hunk 1]
```diff
@@ -95,7 +95,7 @@ Directory: 6.11/wheezy
 
 Tags: 4.8.4, 4.8, 4, argon
 Architectures: amd64, ppc64le
-GitCommit: 9c25cbe93f9108fd1e506d14228afe4a3d04108f
+GitCommit: d001b4cfef7e2d5fede74ce3c6f5e356905ff09c
```
Comment (SimenB) [inline library/node]: This is changed because of an added `architectures` file to _not_ get s390x for 4.8. No actual changes

**Comment by tianon** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index 23138fc..d33385d 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -1,12 +1,16 @@
 node:7 @ amd64
 node:7 @ ppc64le
+node:7 @ s390x
 node:7-alpine @ amd64
 node:7-onbuild @ amd64
 node:7-onbuild @ ppc64le
+node:7-onbuild @ s390x
 node:7-slim @ amd64
 node:7-slim @ ppc64le
+node:7-slim @ s390x
 node:7-stretch @ amd64
 node:7-stretch @ ppc64le
+node:7-stretch @ s390x
 node:7-wheezy @ amd64
 node:alpine @ amd64
 node:argon @ amd64
@@ -21,20 +25,28 @@ node:argon-stretch @ ppc64le
 node:argon-wheezy @ amd64
 node:boron @ amd64
 node:boron @ ppc64le
+node:boron @ s390x
 node:boron-alpine @ amd64
 node:boron-onbuild @ amd64
 node:boron-onbuild @ ppc64le
+node:boron-onbuild @ s390x
 node:boron-slim @ amd64
 node:boron-slim @ ppc64le
+node:boron-slim @ s390x
 node:boron-stretch @ amd64
 node:boron-stretch @ ppc64le
+node:boron-stretch @ s390x
 node:boron-wheezy @ amd64
 node:latest @ amd64
 node:latest @ ppc64le
+node:latest @ s390x
 node:onbuild @ amd64
 node:onbuild @ ppc64le
+node:onbuild @ s390x
 node:slim @ amd64
 node:slim @ ppc64le
+node:slim @ s390x
 node:stretch @ amd64
 node:stretch @ ppc64le
+node:stretch @ s390x
 node:wheezy @ amd64
````

</details>

**Comment by tianon** [comment]
LGTM (merging, since no changes except a new arch)

**Comment by tianon** [comment]
Lol, I think something needs to be re-generated from the template to pick up the `case` statement changes:
```console
Building bashbrew/cache:86b52efe24cc9273a60ea31dd75abcf62b96cb86d0a62a747c9b6b81bea6815e (node:8.2.1)
error: failed building "node" (tags "8.2.1, 8.2, 8, latest")
exit status 1
docker ["build" "-t" "bashbrew/cache:86b52efe24cc9273a60ea31dd75abcf62b96cb86d0a62a747c9b6b81bea6815e" "--rm" "--force-rm" "-"] output:
Sending build context to Docker daemon  30.72kB

Step 1/9 : FROM buildpack-deps:jessie
 ---> 75dd44ac0b69
Step 2/9 : RUN groupadd --gid 1000 node   && useradd --uid 1000 --gid node --shell /bin/bash --create-home node
 ---> Using cache
 ---> 0dd16390c87d
Step 3/9 : RUN set -ex   && for key in     9554F04D7259F04124DE6B476D5A82AC7E37093B     94AE36675C464D64BAFA68DD7434390BDBE9B9C5     FD3A5288F042B6850C66B31F09FE44734EB7990E     71DCFD284A79C3B38668286BC97EC7A07EDE3FC1     DD8F2338BAE7501E3DD5AC78C273792F7D83545D     B9AE9905FFD7803F25714661B63B535A4C206CA9     C4F0DFFF4E8C1A8236409D08E73BC641CC11F4C8     56730D5401028683275BD23C23EFEFE93C4CFFFE   ; do     gpg --keyserver pgp.mit.edu --recv-keys "$key" ||     gpg --keyserver keyserver.pgp.com --recv-keys "$key" ||     gpg --keyserver ha.pool.sks-keyservers.net --recv-keys "$key" ;   done
 ---> Using cache
 ---> 5bb046bfcd72
Step 4/9 : ENV NPM_CONFIG_LOGLEVEL info
 ---> Using cache
 ---> db52a7cd887b
Step 5/9 : ENV NODE_VERSION 8.2.1
 ---> Using cache
 ---> 0ea2318494fa
Step 6/9 : RUN ARCH= && dpkgArch="$(dpkg --print-architecture)"   && case "${dpkgArch##*-}" in     amd64) ARCH='x64';;     ppc64el) ARCH='ppc64le';;     *) echo "unsupported architecture"; exit 1 ;;   esac   && curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-$ARCH.tar.xz"   && curl -SLO --compressed "https://nodejs.org/dist/v$NODE_VERSION/SHASUMS256.txt.asc"   && gpg --batch --decrypt --output SHASUMS256.txt SHASUMS256.txt.asc   && grep " node-v$NODE_VERSION-linux-$ARCH.tar.xz\$" SHASUMS256.txt | sha256sum -c -   && tar -xJf "node-v$NODE_VERSION-linux-$ARCH.tar.xz" -C /usr/local --strip-components=1   && rm "node-v$NODE_VERSION-linux-$ARCH.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt   && ln -s /usr/local/bin/node /usr/local/bin/nodejs
 ---> Running in d76b91c21281
unsupported architecture
Removing intermediate container d76b91c21281
The command '/bin/sh -c ARCH= && dpkgArch="$(dpkg --print-architecture)"   && case "${dpkgArch##*-}" in     amd64) ARCH='x64';;     ppc64el) ARCH='ppc64le';;     *) echo "unsupported architecture"; exit 1 ;;   esac   && curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-$ARCH.tar.xz"   && curl -SLO --compressed "https://nodejs.org/dist/v$NODE_VERSION/SHASUMS256.txt.asc"   && gpg --batch --decrypt --output SHASUMS256.txt SHASUMS256.txt.asc   && grep " node-v$NODE_VERSION-linux-$ARCH.tar.xz\$" SHASUMS256.txt | sha256sum -c -   && tar -xJf "node-v$NODE_VERSION-linux-$ARCH.tar.xz" -C /usr/local --strip-components=1   && rm "node-v$NODE_VERSION-linux-$ARCH.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt   && ln -s /usr/local/bin/node /usr/local/bin/nodejs' returned a non-zero code: 1
```

**Comment by SimenB** [comment]
Bah, I'm dumb... https://github.com/nodejs/docker-node/pull/498

Needs to wait for a review

**Comment by tianon** [comment]
No problem, thanks for taking care of it!

**Comment by SimenB** [comment]
Updated PR: #3302

**Pull request merged**